.github/workflows/check-alpine-version.yml

```diff
@@ -0,0 +1,68 @@
+name: Check Supported Alpine Version
+on:
+  schedule:
+    - cron: "0 17 * * *"
+  workflow_dispatch:
+  push:
+
+jobs:
+  create-prs:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch: ["v7", "v8"]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Get current Alpine version
+        id: alpine-current
+        run: |
+          version=$(sed -n 's/^FROM \(.*\)/\1/p' Dockerfile.goreleaser)
+          echo "version=$version" >> $GITHUB_OUTPUT
+      - name: Get latest Alpine tag semantic version, not 'latest'
+        id: alpine-latest
+        run: |
+          version=$(curl -s https://hub.docker.com/v2/repositories/library/alpine/tags/?page_size=100 | jq -r '.results[].name' | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V | tail -n 1)
+          echo "version=$version" >> $GITHUB_OUTPUT
+
+      - name: Update from ${{ steps.alpine-current.outputs.version }} to alpine:${{ steps.alpine-latest.outputs.version }}
+        id: update-alpine
+        if: steps.alpine-current.outputs.version != steps.alpine-latest.outputs.version
+        run: ./scripts/update-alpine-version.sh ${{ steps.alpine-latest.outputs.version }}
+
+      - name: Create pull request
+        if: steps.update-alpine.outcome == 'success'
+        uses: peter-evans/create-pull-request@v6
```

Comment:
We should probably be using a SHA for this one instead of v6.

Reply:
Because it's external in general or something about the specific action? (The Go version action and Trivy scan also use tags)

Reply:
Effectively because it is smaller. `peter-evans` versus github/microsoft, etc.